src/DynamoCore/Graph/Workspaces/SerializationExtensions.cs

```diff
@@ -95,5 +99,214 @@ internal static string SerializeIntegerSliderAs32BitType(string json)
 
             return rgx2.Replace(result, "CoreNodeModels.Input.IntegerSlider");
         }
+
+        internal static string keyProperty = "ConcreteType";
+
+        /// <summary>
+        /// Orders the properties of each Node according to the specify in the NodePropertiesOrder function
+        /// </summary>
+        /// <param name="metaContent"></param>
+        /// <returns></returns>
+        internal static string customOrderNodes(string metaContent)
+        {
+            try
+            {
+                string metaNodesStartKey = "\"Nodes\": [\r\n   ";
+                string metaNodesEndKey = "\r\n  ]";
+                int flatNodesStartIndex = metaContent.IndexOf(metaNodesStartKey) + metaNodesStartKey.Length;
```

Comment:
I think this code has the potential to be very slow, have you profiled the serialization performance of a very large graph before and after this change?